packages/compass-e2e-tests/tests/atlas-login.test.ts

```diff
@@ -121,6 +125,10 @@ describe('Atlas Login', function () {
   });
 
   after(async function () {
+    if (TEST_COMPASS_WEB) {
```

Comment:
Unfortunately mocha still calls after() if you this.skip() in before() and that's by design: https://github.com/mochajs/mocha/issues/3120